stories/index.stories.tsx

```diff
@@ -82,3 +91,130 @@ storiesOf('Markers', module).add('adding a marker', () => (
     />
   </Map>
 ))
+
+storiesOf('Overlays', module)
+  .add('Circles', () => (
+    <Map
+      tokenOrCallback={devToken}
+      region={{
+        latitude: 47.6754,
+        longitude: -122.2084,
+        latitudeSpan: 0.006,
+        longitudeSpan: 0.006,
+      }}
+    >
+      <CircleOverlay
+        latitude={47.6754}
+        longitude={-122.2084}
+        radius={100}
+        styles={{ fillColor: 'red', strokeColor: 'red' }}
+      />
+    </Map>
+  ))
+  .add('Polygons', () => (
+    <Map
+      tokenOrCallback={devToken}
+      region={{
+        latitude: 47.6754,
+        longitude: -122.2084,
+        latitudeSpan: 0.006,
+        longitudeSpan: 0.006,
+      }}
+    >
+      <PolygonOverlay
+        points={[
+          [47.6754, -122.2084],
+          [47.6764, -122.2073],
+          [47.6747, -122.2061],
+        ]}
+      />
+    </Map>
+  ))
+  .add('Polyline', () => (
+    <Map
+      tokenOrCallback={devToken}
+      region={{
+        latitude: 47.6754,
+        longitude: -122.2084,
+        latitudeSpan: 0.006,
+        longitudeSpan: 0.006,
+      }}
+    >
+      <PolylineOverlay
+        points={[
+          [47.6754, -122.2084],
+          [47.6764, -122.2073],
+        ]}
+        styles={{ lineWidth: 4 }}
+      />
+    </Map>
+  ))
+
+storiesOf('Directions', module)
+  .add('Driving', () => (
+    <Map
+      tokenOrCallback={devToken}
+      region={{
+        latitude: 47.6754,
+        longitude: -122.2084,
+        latitudeSpan: 0.006,
+        longitudeSpan: 0.006,
+      }}
+    >
+      <Directions
+        origin={[47.6754, -122.2084]}
+        destination={[47.68, -122.2135]}
+        transportType="Automobile"
+        style={{ lineWidth: 3 }}
+      />
+    </Map>
+  ))
+  .add('Walking', () => (
+    <Map
+      tokenOrCallback={devToken}
+      region={{
+        latitude: 47.6754,
+        longitude: -122.2084,
+        latitudeSpan: 0.006,
+        longitudeSpan: 0.006,
+      }}
+    >
+      <Directions
+        origin={[47.6754, -122.2084]}
+        destination={[47.68, -122.2135]}
+        transportType="Walking"
+        style={{ lineWidth: 3 }}
+      />
+    </Map>
+  ))
+  .add('Getting route details', () => {
+    const [time, setTime] = useState(0)
```

Comment:
this won't work in storybook, you can either create this component outside of the story and use it in the story, or we can wait for #31 to drop before adding this story.

Reply:
I've modified this within my next push to add a components.stories.tsx which includes a RouteDetailExample component which is refactored from this